Editorial refresh (#36)
w3c-ccg/did-pkh: did-pkh-method-draft.md

standards-conformant way. This "DID-wrapping" of an existing identifier can be
used in combination with other DID-compatible technologies, such as W3C
Verifiable Credentials or Authorization Capabilities, and produce proper
signature-suite definitions, such as "metamask-signing" (off-chain signing by

> externally-owned accounts, i.e. personal wallet, according to the [eip712][]

Reviewer: ```suggestion
externally-owned accounts, i.e., personal wallets, according to the [eip712][]
```

  blockchain-published DID document according to another method (such as `did:tz`,
  `did:btcr`, or `did:ethr`), its DID document can be translated to the form
  of that method's documents, and it can be registered there.
- if a web application or dapp can interact with a blockchain wallet to produce
  deterministic, signed receipts of `did:pkh` consent/authorization events such

>   those specified by the Sign-In With Ethereum ([SIWE][]) or Sign-In With

Reviewer: ```suggestion
  as those specified by the Sign-In With Ethereum ([SIWE][]) or Sign-In With
```

  Anything ([SIWX][]), such as [CACAO][] objects, these can be used to make that
  `did:pkh` identifier the `controller` of a second DID, given that said second
  DID method supports `controller` patterns.

## Design Goals

1. The primary goal of this method is to allow any valid blockchain address to
   "spin up" a feature-limited but valid and widely interoperable DID and DID
   Document, valid in a limited context where accounts are represented by DIDs. 
2. This method is very narrow and unopinionated to allow a wide range of

>    implementations. For example, some blockchain systems do and some do not

Reviewer: ```suggestion
   implementations. For example, some blockchain systems do — and some do not —
```

   enable deterministic public key "recovery" from a signature and public key
   hash, which leads to very different verification methods across pkh types.
3. For example, the validity of each address to be wrapped in a DID is checked
   according to the [CAIP-10][] specification before generating a DID document,

>    to prevent a did:pkh being presented as valid that would not be on its

Reviewer: ```suggestion
   to prevent a `did:pkh` being presented as valid that would not be on its
```

   corresponding blockchain. **No further validation** is assumed or provided in
   the reference implemention, but implementers may still choose to gate

>    generation to on-chain accounts or balance-holding accounts as per the
>    requirements of their specific use case.

Reviewer: ```suggestion
   generation to on-chain accounts and/or to balance-holding accounts, among
   other tests, according to the requirements of their specific use case.
```
